stellar-common/src/payments.rs

```diff
@@ -87,6 +108,19 @@ fn map_payments(block: Block) -> Result<Payments, substreams::errors::Error> {
                     None => return,
                 }
             }
+            // Unfortunately there are many more ways that accounts can transfer
+            // value. We're actually right in the middle of developing a
+            // holistic processor to enumerate the various transfers that can
+            // occur, which should serve as an excellent guide here [1]. While
+            // that one is concerned with a unified model, for your purposes
+            // you'll want to leverage the detailed explanations there to
+            // capture your substream equivalent here.
+            //
+            // In the "Classic Operations and Events" section, you can see all
+            // of the operations you'll need to include here to encompass all
+            // payments.
+            //
+            // https://github.com/stellar/go/issues/5580
```

Comment:
Perfect, will check the issue and start adding parts here

Reply:
In the case of a simple payment, how can I know if an account is an issuer or not? I would need that information (if possible) in the block itself. 

Substreams do not allow you to query at runtime from a data source if the account is an issuer or not.

Reply:
The asset details themselves hold the issuer, so if the source/destination matches the asset issuer then you know it's a mint/burn.